public/js/src/bridge/bridge_command.tag

```diff
@@ -613,6 +625,8 @@
             view.shipit = config.shipit_url + 'v1/shipment/' + shipment + '/environment/' + environment;
             view.erp = 'http://erp.services.dmtio.net/products/' + shipment + '/' + environment;
             view.currentRoute = '#bridge/' + shipment + '/' + environment;
+            view.shipmentStatus = 'http://helmit.services.dmtio.net/shipment/status/:barge/' + shipment + '/' + environment;
+            view.shipmentEvents = 'http://helmit.services.dmtio.net/shipment/events/:barge/' + shipment + '/' + environment;
```

Comment:
These url values should be pulled from the config. We set a config on the window with all the urls from the server so they can be passed in via envVars.